ee/vulnerability-dashboard/scripts/update-reports.js

```
@@ -274,7 +274,7 @@ module.exports = {
     await sails.helpers.flow.until(async ()=>{
       // * * *
       // Load a page of vulnerable software versions.
-      // [?] https://fleetdm.com/docs/rest-api/rest-api#list-software-versions
+      // [?] https://fleetdm.com/docs/api/rest-api#list-software-versions
```

ESLint

[error] Insert `··`

(prettier/prettier)